lib/content-services/src/lib/search/services/search-facet-filters.service.ts

```diff
@@ -335,6 +342,27 @@ export class SearchFacetFiltersService implements OnDestroy {
         };
     }
 
+    private loadCategoryNames(bucketList: FacetFieldBucket[]) {
+        bucketList.forEach((item) => {
+            const categoryId = item.label.split('/').pop();
+            this.categoryService.getCategory(categoryId)
+                .pipe(
+                    concatMap((categoryEntry) => this.categoryService.searchCategories(categoryEntry.entry.name)),
```

Comment:
Are you using the `searchCategories` just to get the same category as `getCategory` returns but with the `path`? If so you don't need it anymore, my JS-API PR adds support for including the `path` in most categories endpoints so you can just include `path` adding `{include: ['path']}` as option in `getCategory` call as soon as my JS-API PR will be merged

Reply:
right, that is how it was planned)